pkg/model/config/config_test.go

```diff
@@ -31,6 +31,45 @@ var _ = Describe("PluginConfig", func() {
 		Data2 string `json:"data-2"`
 	}
 
+	resource := ResourceData{Group: "Foo", Kind: "Baz", Version: "v1"}
+	resource.Webhooks = &Webhooks{false, false, true}
+
+	It("should merge webhook when webhook is NOT equals", func() {
+		otherResource := ResourceData{Group: "Foo", Kind: "Baz", Version: "v1"}
+		otherResource.Webhooks = &Webhooks{false, true, false}
+		resource.merge(otherResource)
+		Expect(resource.Webhooks.Defaulting).To(BeFalse())
+		Expect(resource.Webhooks.Validation).To(BeTrue())
+		Expect(resource.Webhooks.Conversion).To(BeTrue())
+	})
+
+	It("should merge not update the conversion and validation to false when it is true already", func() {
+		otherResource := ResourceData{Group: "Foo", Kind: "Baz", Version: "v1"}
+		otherResource.Webhooks = &Webhooks{false, false, false}
+		resource.merge(otherResource)
+		Expect(resource.Webhooks.Defaulting).To(BeFalse())
+		Expect(resource.Webhooks.Validation).To(BeTrue())
```

Comment:
```suggestion
		Expect(resource.Webhooks.Validation).To(BeFalse())
```

Reply:
It needs to be true. Note that we updated this value in the test above.

Reply:
You are right. I don't know why I expected resource to be reseted. You can resolve this comment.